app/src/main/res/values/strings.xml

```diff
@@ -15,10 +15,12 @@ to your home internet.</string>
     <string name="checking_with_home_assistant">Checking with Home Assistant</string>
     <string name="skip">Skip</string>
     <string name="retry">Retry</string>
+    <string name="ok">Ok</string>
     <string name="attempting_registration">Registering application…</string>
     <string name="unable_to_register">Unable to Register Application</string>
     <string name="error_with_registration">Please check to ensure you have the mobile_app
 integration enabled on your home assistant instance.</string>
+    <string name="error_connection_failed">Unable to connect to Home Assistant, check your URL.</string>
```

Comment:
Can we rename this to include onboarding. We wouldn't want to use the "check your URL" part when we can't connect after authentication is done. 

```suggestion
    <string name="error_onboarding_connection_failed">Unable to connect to Home Assistant, check your URL.</string>
```

Reply:
Fixed!